.ci/integration_test

```diff
@@ -240,10 +240,10 @@ function run_test_on_cluster() {
     setup-aws-infrastructure
   fi
 
-  export ETCD_VERSION=${ETCD_VERSION:-"v3.4.13"}
+  export ETCD_VERSION=${ETCD_VERSION:-"v3.4.13-bootstrap"}
   echo "Etcd version: ${ETCD_VERSION}"
 
-  export ETCDBR_VERSION=${ETCDBR_VERSION:-${ETCDBR_VER:-"v0.11.1"}}
+  export ETCDBR_VERSION=${ETCDBR_VERSION:-${ETCDBR_VER:-"v0.12.0-dev-2cb339762d03d5cc9c78bda855fdf48f309de9c3"}}
```

Comment:
```suggestion
  export ETCDBR_VERSION=${ETCDBR_VERSION:-${ETCDBR_VER:-"v0.12.0"}}
```

Reply:
I had initially set this image version to test the running of cluster-based integration test with the new chart within the PR. I will change it now to `v0.12.0`, since the CICD pipeline injects `EFFECTIVE_VERSION` into the `ETCDBR_VER` variable, so TM tests should run fine. But if you plan on running cluster-based integration tests locally for this PR, please also pass `ETCDBR_VERSION` env var to the script.